docs/document-creator/config-file.md

```diff
@@ -263,6 +264,20 @@ By default, if this field is not present, it will create the document with "tt"
 
 _i.e. "document-1.tt"_
 
+### `"fileName"` field
+
+```json
+"fileName": "ebl-<%= ebl-number %>",
+```
+
+_Note: This fileName field is optional. Only add this section in if you want to customise the file name to follow the form fields when uploading data file._
+
+The `"fileName"` field is a string that refers to the file name of the created document.
+
+By default, if this field is not present, The documents will follow the forms name with increment number.
+
+_i.e. "document-1"_
+
```

Comment:
ok gd! but can further elaborate on the <%= abc %>,
if can, also provide link for user to learn more about it.